packages/editor/src/plugins/image-gallery/components/image-add-menu.tsx

```diff
@@ -0,0 +1,196 @@
+import { EditorTooltip } from '@editor/editor-ui/editor-tooltip'
```

Comment:
This file is very similar to ImageSelectionScreen in Image plugin, they will be combined in some way